packages/gatsby/src/commands/build.ts

```diff
@@ -70,8 +70,8 @@ module.exports = async function build(program: IBuildArgs): Promise<void> {
   const buildSpan = buildActivity.span
   buildSpan.setTag(`directory`, program.directory)
 
-  const { graphqlRunner: bootstrapGraphQLRunner } = await bootstrap({
-    ...program,
+  const { gatsbyNodeGraphQLFunction } = await bootstrap({
```

Comment:
@ascorbic btw when I mean reducing irrelevant changes out, this is the kind. This file has 5 changed lines. Three of them concern a var rename. I bet we can find many of these that are not really relevant to the core of this PR.

You can probably separate out the bootstrap api changes as well. (Export default to export binding, program no longer being spread, returned keys, etc).